torch_xla/csrc/init_python_bindings.cpp

```diff
@@ -1537,6 +1537,57 @@ void InitXlaModuleBindings(py::module m) {
     }
     return std::string();
   });
+  m.def("_get_local_shards",
+        [](const at::Tensor& input) -> std::vector<at::Tensor> {
+          XLATensorPtr xtensor = bridge::GetXlaTensor(input);
+          XLA_CHECK(xtensor->GetXlaData() != nullptr)
```

Comment:
This may be redundant, but shall we also check for `XLA_USE_SPMD=1`?

Reply:
Good point, I'll add that check. I'll also enforce that `sharding_spec()` is non-null, since otherwise we'll return a single shard wrapping the tensor's data.